bip39-lib/src/nonJvmMain/kotlin/cash/z/ecc/android/crypto/Pbkdf2Sha512.kt

```diff
@@ -33,23 +36,20 @@ internal actual object Pbkdf2Sha512 {
      * @param dkLen the key length in bits
      */
     actual fun derive(p: CharArray, s: ByteArray, c: Int, dkLen: Int): ByteArray {
-        Buffer().use { baos ->
-            val dkLenBytes = dkLen / 8
-            val pBytes = p.foldIndexed(ByteArray(p.size)) { i, acc, c ->
-                acc.apply { this[i] = c.code.toByte() }
-            }
-            val hLen = 20.0
-            // note: dropped length check because it's redundant, given the size of an int in kotlin
-            val l = ceil(dkLenBytes / hLen).toInt()
-            for (i in 1..l) {
-                f(pBytes, s, c, i).let { Tn ->
-                    baos.write(Tn)
-                }
-            }
-            return ByteArray(dkLenBytes).apply {
-                baos.read(this)
+        val dkLenBytes = dkLen / 8
```

Comment:
Thanks for rolling back use of Okio here.  This minimizes the surface area of using third parties, and will make it easier to swap out Okio with something else in the future since its now just the hashing functions.